workflow/controller/operator_metrics_test.go

```diff
@@ -237,7 +238,11 @@ func getMetricStringValue(metric prometheus.Metric) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("%v", metricString.String()), nil
+
+	// Workaround for https://github.com/prometheus/client_model/issues/83
+	space := regexp.MustCompile(`\s+`)
```

Comment:
small optimization: this can be moved outside of the function as it's a constant and this function is called multiple times

Reply:
Fixed